content/docs/user-guide/components/reference/gradients/image-gradient.md

```diff
@@ -5,13 +5,13 @@ description: Use the Image Gradient component to generate a gradient from an ima
 ---
 
 Add the **Image Gradient** component to generate a gradient from an image. The component also provides advanced
-configuration options, such as tiling, scaling, and custom sampling types.
+configuration options, such as tiling, scaling, and custom sampling types. Image creation and editing are also supported.
```

Comment:
You don't have to fix this, but there is a whitespace issue on lines 7-8.  I wanted to suggest deleting the first occurrence of "also", but I'm unable to.